packages/arb-token-bridge-ui/src/hooks/useBalance.ts

```diff
@@ -92,8 +92,11 @@ const useBalance = ({ chainId, walletAddress }: UseBalanceProps) => {
       } catch (error) {
         // log some extra info on sentry in case multi-caller fails
         Sentry.configureScope(function (scope) {
-          scope.setExtra('token_addresses', addresses)
-          Sentry.captureException(error)
+          // tags only allow primitive values
```

Comment:
I see a lot of repetitive code. We should abstract configuring all the extra-data for Sentry in a common function like we did for `isUserRejectedError`.

We can convert this PR to `feat` but it'd be great if we have something like - 

```
function captureErrorWithData( 
{ 
  error: Error,
  originFunction, 
  token?:string,
  chainId:number,
  ... and any other keys that we use with `setTags` in our codebase 
} )
```

We will benefit from it also because it will also consolidate any different tag names (if any) we might be using for the same data.